.github/ISSUE_TEMPLATE/WG_member_request.yaml

```diff
@@ -0,0 +1,62 @@
+name: Working Group Member Request
+description: Request membership to serve on a PowerShell Working Group
+title: Working Group Member Request
+labels: [WG-NeedsReview, WG-Cmdlets, WG-Engine, WG-Interactive-Console, WG-Remoting, Needs-Triage]
+body:
+- type: markdown
+  attributes:
+    value: |
+      ## Thank you for your interest in joining a PowerShell Working Group. Please complete the following public form to request membership to a PowerShell Working Group.
+- type : dropdown
+  id : request_type
+  validations:
+    required: true
+  attributes:
+    label: Name of Working Group you are requesting to join?
+    description: >-
+      Please select the name of the working group you are requesting to join. (Select one)
```

Comment:
Maybe add some text that WGs not in the list are not currently accepting new members

Reply:
Note: Not all Working Groups are accepting new members at this time.